src/js/components/sorter/Sorter.jsx

```diff
@@ -8,19 +8,23 @@ class Sorter extends Component {
     sortby: this.props.sortby
   };
 
-  clickHandler = () => {
-    if (event.target.value) {
-      this.setState({sortby: event.target.value});
-      this.props.changeHandler(this.state);
+  changeHandler = () => {
```

Comment:
here same as from `SearchFilter.jsx` 
you need to be carefull when accessing global variable - this time you had a lot of luck that it's was providin ~good value